sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -2527,9 +2549,17 @@ glsTextureTestUtil.computeTextureLookupDiff2DArray = function(result, reference,
 
         for (var px = 0; px < result.getWidth(); px++) {
             /** @type {Array<number>} */
-            var resPix = deMath.divide(deMath.subtract(result.getPixel(px, py), sampleParams.colorBias), sampleParams.colorScale);
+            resPix[0] = (resPix[0] - sampleParams.colorBias[0]) / sampleParams.colorScale[0];
```

Comment:
You forget to declare resPix and initialize it with result.getPixel(px, py) here.